app/routes/session.py

```diff
@@ -123,6 +123,8 @@ def login():
             "account_service_log_out_url"
         )
 
+    cookie_session["preview_questions"] = g.schema.json.get("preview_questions", False)
```

Comment:
Why is this needed?

Reply:
I need it in template helpers, is there another way of extracting  it in template helpers since there is no concept of schema there?

Reply:
Should we load schema from metadata in template helpers to get that field?

Reply:
Ya we should get the value from schema instead of the cookie.

Reply:
Ok, getting value from schema now.